client/src/app/site/motions/components/motion-list/motion-list.component.ts

```diff
@@ -166,4 +169,52 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion> imple
             this.translate.instant('Motions') + '.csv'
         );
     }
+
+    /**
+     * Deletes the items selected.
+     * SelectedRows is only filled with data in multiSelect mode
+     */
+    public deleteSelected(): void {
+        this.selectedRows.forEach(vm => {
+            this.repo.delete(vm);
```

Comment:
this can delete one motion for me, but deleting two or more motions thwos an error.
You might need to use await or .then here.

```
Error: Uncaught (in promise): Error: A server error occured. Please contact your system administrator.
```